src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();

```

Comment:
For the next iteration you are probably better off using `screen`.
You import it from the testing-library, just as `render` and it will allow you to clean this up like this:

```suggestion
render(<App />);
  const linkElement = screen.getByText(/learn react/i);
  expect(linkElement).toBeInTheDocument();

```

This especially useful when you have to use multiple queries in a single test.